neo/Prompt/Commands/Bootstrap.py

```diff
@@ -7,6 +7,9 @@
 import shutil
 import os
 
+DEFAULT_TMP_BOOTSTRAP_FILE = os.path.join(settings.DATA_DIR_PATH,
```

Comment:
Move this const into the function, because the settings data for might be changed using the --datadir argument, which would not be used like this